test-suite/test_tools.cc

```diff
@@ -11,14 +11,6 @@
 
 #include "squid.h"
 #include "dlink.h"
-#include <iostream>
```

Comment:
These iostream and xassert() removals do not seem relevant.  Please split into a separate PR or document in the description how its related.